packages/devextreme/js/ui/html_editor.d.ts

```diff
@@ -46,7 +46,162 @@ export type HtmlEditorImageUploadTab = 'url' | 'file';
 /** @public */
 export type HtmlEditorPredefinedContextMenuItem = 'background' | 'bold' | 'color' | 'font' | 'italic' | 'link' | 'image' | 'strike' | 'subscript' | 'superscript' | 'underline' | 'blockquote' | 'increaseIndent' | 'decreaseIndent' | 'orderedList' | 'bulletList' | 'alignLeft' | 'alignCenter' | 'alignRight' | 'alignJustify' | 'codeBlock' | 'variable' | 'undo' | 'redo' | 'clear' | 'insertTable' | 'insertHeaderRow' | 'insertRowAbove' | 'insertRowBelow' | 'insertColumnLeft' | 'insertColumnRight' | 'deleteColumn' | 'deleteRow' | 'deleteTable' | 'cellProperties' | 'tableProperties';
 /** @public */
-export type HtmlEditorPredefinedToolbarItem = 'background' | 'bold' | 'color' | 'font' | 'italic' | 'link' | 'image' | 'size' | 'strike' | 'subscript' | 'superscript' | 'underline' | 'blockquote' | 'header' | 'increaseIndent' | 'decreaseIndent' | 'orderedList' | 'bulletList' | 'alignLeft' | 'alignCenter' | 'alignRight' | 'alignJustify' | 'codeBlock' | 'variable' | 'separator' | 'undo' | 'redo' | 'clear' | 'cellProperties' | 'tableProperties' | 'insertTable' | 'insertHeaderRow' | 'insertRowAbove' | 'insertRowBelow' | 'insertColumnLeft' | 'insertColumnRight' | 'deleteColumn' | 'deleteRow' | 'deleteTable';
+export type HtmlEditorPredefinedToolbarItem = 'background' | 'bold' | 'color' | 'font' | 'italic' | 'link' | 'image' | 'size' | 'strike' | 'subscript' | 'superscript' | 'underline' | 'blockquote' | 'header' | 'increaseIndent' | 'decreaseIndent' | 'orderedList' | 'bulletList' | 'alignLeft' | 'alignCenter' | 'alignRight' | 'alignJustify' | 'codeBlock' | 'variable' | 'separator' | 'undo' | 'redo' | 'clear' | 'cellProperties' | 'tableProperties' | 'insertTable' | 'insertHeaderRow' | 'insertRowAbove' | 'insertRowBelow' | 'insertColumnLeft' | 'insertColumnRight' | 'deleteColumn' | 'deleteRow' | 'deleteTable' | 'ai';
+
+/** @public */
+export type HtmlEditorAICommandName = 'summarize' | 'proofread' | 'expand' | 'shorten' | 'changeStyle' | 'changeTone' | 'translate' | 'askAI';
+
+/** @public */
+export type HtmlEditorAIChangeStyleOption =
+    | 'formal'
+    | 'informal'
+    | 'technical'
+    | 'business'
+    | 'creative'
+    | 'journalistic'
+    | 'academic'
+    | 'persuasive'
+    | 'narrative'
+    | 'expository'
+    | 'descriptive'
+    | 'conversational'
+    | string;
```

Comment:
```suggestion
    | 'conversational';
```

I think if we remove `string` out of the these types it will make it more convenient to reusethem  in end projects.

Assuming that string will go here:

```ts
export type HtmlEditorAIChangeStyleCommand = HtmlEditorAICommandBase<'changeStyle', (HtmlEditorAIChangeStyleOption | string)[]>;
```

Reply:
Maybe it's also better for reading when surfing through d.ts (I can't see literal values until I go directly to them):


<img width="580" alt="image" src="https://github.com/user-attachments/assets/884d845e-dc02-4ec8-8527-c55e4aeb1c67" />